includes/payment-methods/class-wc-stripe-upe-payment-gateway.php

```
@@ -1129,22 +1134,18 @@ public function add_currency_conversion_notice( WC_Order $order ): void {
 		// Use the decimal count for the presentment currency, not the store's price decimal
 		// setting, to avoid incorrect rounding (e.g. JPY stores with 0 decimal places).
-		$presentment_currency_lower = strtolower( $presentment_currency );
-		$rate_decimals              = 2;
-		if ( in_array( $presentment_currency_lower, WC_Stripe_Helper::no_decimal_currencies(), true ) ) {
-			$rate_decimals = 0;
-		} elseif ( in_array( $presentment_currency_lower, WC_Stripe_Helper::three_decimal_currencies(), true ) ) {
-			$rate_decimals = 3;
-		}
-		$rate_amount = wc_format_decimal( $presentment_amount / $stripe_amount, $rate_decimals );
+		$presentment_currency       = strtolower( $presentment_currency ); // Make sure the original currency code is in lowercase.
+		$presentment_currency_upper = strtoupper( $presentment_currency );
```

PHPMD (2.15.0)

[error] The variable $presentment_currency_upper is not named in camelCase. (undefined)

(CamelCaseVariableName)